docs/resources/cluster.md

```diff
@@ -13,12 +13,13 @@ CockroachDB Cloud cluster.
 ## Example Usage
 
 ```terraform
-resource "cockroach_cluster" "dedicated" {
-  name           = "cockroach-dedicated"
+resource "cockroach_cluster" "advanced" {
+  name           = "cockroach-advanced"
   cloud_provider = "GCP"
+  plan           = "ADVANCED"
   dedicated = {
     storage_gib  = 15
-    machine_type = "n1-standard-4"
+    machine_type = "n2-standard-4"
```

Comment:
https://github.com/cockroachdb/terraform-provider-cockroach/pull/224/files#diff-78accda5330258f3c72bbbd93e54e91c1a00463630f0003901bbda25a7326f21R99, you have number of CPUs I think. Can it be either?

Reply:
Yes, my understanding is that it can be either. I just fixed this b/c I noticed it was using an old machine type that we're phasing out.

Reply:
Do we have a preference on defining the machine type vs the number of CPUs?

Reply:
I'm not sure, referring that to @marksoper.